core/scripts/server.ts

```diff
@@ -21,15 +34,16 @@ const NOT_ALLOWED_SSR_EXTENSIONS_REGEX = new RegExp(`(.*)(${config.server.ssrDis
 const isProd = process.env.NODE_ENV === 'production'
 process['noDeprecation'] = true
 
-const app = express()
+const app = express.default();
 
-serverExtensions.serverModules.forEach(serverModule => {
-  if (Array.isArray(serverModule)) {
-    require(resolve(serverModule[0] + '/server.ts'))(app, serverModule[1])
-  } else {
-    require(resolve(serverModule + '/server.ts'))(app)
-  }
-})
+// TODO: This needs discussion. For now it's commented to have possibility to run a server
+// serverExtensions.serverModules.forEach(serverModule => {
+//   if (Array.isArray(serverModule)) {
+//     require(resolve(serverModule[0] + '/server.ts'))(app, serverModule[1])
```

Comment:
I've commented this code as you can see, TS files are added on runtime which won't work at all in this case. Maybe we need to create a plugin for webpack to resolve it during the building process?